src/main/java/com/redhat/cloud/policies/app/rest/AdminService.java

```diff
@@ -173,36 +172,4 @@ private void putToBucket(Map<String, Integer> buckets, int count) {
         buckets.computeIfPresent(bucket, (b, c) -> buckets.get(b) + 1);
         buckets.putIfAbsent(bucket, 1);
     }
-
-    // Trigger Tenant Tuple
-    public static class TTT {
```

Comment:
Not used anymore? or is there an OrgId replacement?

Reply:
I think it stopped being used when policies was refactored. No link with the org ID migration.